frontend/src/collapsible.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * sri sri guru gaurangau jayatah
+ */
+
+window.collapsible = () => {
+  return {
+    expanded: false,
+
+    toggle() {
+      this.expanded = !this.expanded;
+    },
+    itemsList: {
```

Comment:
We can shorten this to `contents`, because in the context of the `collapsible` component we should not assume what type of content will be there: items or something else.